addon/components/ce/content-editable.js

```diff
@@ -159,7 +160,8 @@ export default class ContentEditable extends Component {
                                    new IgnoreModifiersHandler({rawEditor}),
                                    new UndoHandler({rawEditor}),
                                    new BoldItalicUnderlineHandler({rawEditor}),
-                                   new FallbackInputHandler({rawEditor})
+                                   new FallbackInputHandler({rawEditor}),
```

Comment:
By not being the last one in the list, will it still full fill what it says it does?